src/platform/unix/ushellcontextmenu.pas

```diff
@@ -691,7 +695,6 @@ procedure TShellContextMenu.CreateActionSubMenu(MenuWhereToAdd:TComponent; aFile
   // Add shortcut to launch file association cnfiguration screen
   if gIncludeFileAssociation then
   begin
-    LocalInsertMenuSeparator;
```

Comment:
Why do you remove this?

Reply:
Oh, that's my mistake. I got a little confused with the separators. For some reason, I decided to separate all the Extended options items with one delimiter. 😵 I put it back.